_reports/saksham.md

```diff
@@ -0,0 +1,269 @@
+      - details: >
+         Use appveyor template for coala-utils
+  - 22:
+      - repo: coala-bears
+      - link: https://github.com/coala/coala-bears/commit/c35a2ed04935fd7665514e6af7828f1bfd931a03
+      - details: >
+         Improve and cleanup script for generating bear requirements and generate cabal requirements
+  - 23:
+      - repo: coala-bears
+      - link: https://github.com/coala/coala-bears/commit/26b696969dd38ca1a1ab91163e0154f6c905e252
+      - details: >
+         Move the generate_package script from the coala-bears package to .ci directory
+  - 24:
+      - repo: coala-bears
+      - link: https://github.com/coala/coala-bears/commit/efdfbc1cec98db76a5bb2c75fbfa414686be4db8
+      - details: >
+         Use CabalRequirement for HaskellLintBear
+  - 25:
+      - repo: coala-bears
+      - link: https://github.com/coala/coala-bears/commit/e62d16f8315e4dcd10c9c57af3ab3e5973b10d73
+      - details: >
+         Add language setting for ESLintBear using various plugins for eslint
+  - 26:
+      - repo: coala-bears
+      - link: https://gitlab.com/coala/package_manager/commit/a786450e5cdfd3172caf86c60786c0d1ce6ee10f
+      - details: >
+         Add GherkinLintBear
+  - 27:
+      - repo: coala-bears
+      - link: https://gitlab.com/coala/package_manager/commit/868afe0dcdf6ab0c964f1ce72152fedb132f44cb
+      - details: >
+         Add BashateBear
+  - 28:
+      - repo: dependency_management
+      - link: https://gitlab.com/coala/package_manager/commit/75b3c37f7ffd7b6c9f693d131bf4f627fd95373d
+      - details: >
+         Add PlatformRequirement class for system specific package managers
+  - 29:
+      - repo: dependency_management
+      - link: https://gitlab.com/coala/package_manager/commit/a786450e5cdfd3172caf86c60786c0d1ce6ee10f
+      - details: >
+         Return exitcode after running the install command
+  - 30:
+      - repo: dependency_management
+      - link: https://gitlab.com/coala/package_manager/commit/868afe0dcdf6ab0c964f1ce72152fedb132f44cb
+      - details: >
+         Add CabalRequirement for backwards compatibility
+  - 31:
+      - repo: dependency_management
+      - link: https://gitlab.com/coala/package_manager/commit/995b50ba1861296d333442be9e16ba2b1c4a47b6
+      - details: >
+         Run pip as python module
+  - 32:
+      - repo: bear-install-playbook
+      - link: https://gitlab.com/coala/bear-install-playbook/commit/7ba1c3ef9c29dc58a1c5b0109328317f3b0924df
+      - details: >
+         Add playbook for installing individual bears
+  - 33:
+      - repo: bear-install-playbook
+      - link: https://gitlab.com/coala/bear-install-playbook/commit/1b288844aa157ff4e8c80a3179c5d58163a2719f
+      - details: >
+         Test playbook using docker + Travic CI
+  - 34:
+      - repo: bear-install-playbook
+      - link: https://gitlab.com/coala/bear-install-playbook/commit/7dee7591927c0883269f5741102b90bfb93f84ff
+      - details: >
+         Add instructions on using playbook
+  - 35:
```

Comment:
59 patches vs 35 work activity entries.

I see some patches were ust sync' but I easily see some real work patches which are not included here

Reply:
the others were really very small to include

Reply:
alright added some more

Reply:
Small is ok, as long as it isnt automated or silly.